site/profile/manifests/cvmfs.pp

```diff
@@ -48,6 +49,15 @@
     require => Package['cvmfs']
   }
 
+  file { 'etc/cvfms/config.d':
+    ensure  => 'directory',
+    content => epp('profile/cvmfs/config.d', {
```

Comment:
You cannot define a directory AND a file with the same resource. You will need to define a separate resource with a path and filename that puts it in `/etc/cvmfs/config.d`. The filename could be for example `alien_cache.conf` and based on the template `profile/cvmfs/alien_cache.conf.epp`.

Reply:
Here the directory name suggested `alien_cache.conf` will be the same as the template(.epp)